### PR TITLE
Fixed correct sample frequency.

### PR DESCRIPTION
**Pull request by emilwestergren**
Fix bug where sample delay was sent as nanoseconds to a function expecting microseconds.
@sebastianbaginski peer review

### CODE REVIEW

File: vidhance/sensors/SensorReader.cpp [hunk 1]
```diff
@@ -84,7 +83,7 @@ std::vector<ASensorEvent> SensorReader::getEvents(vidhance_date_time_t timestamp
 	return result;
 }
 vidhance_time_span_t SensorReader::getSamplePeriod() {
-	return vidhance_time_span_new(this->sampleRate / 100);
+	return this->samplePeriod;
```
Comment (sebastianbaginski) [inline vidhance/sensors/SensorReader.cpp]: This method looks const to me

File: vidhance/sensors/SensorReader.cpp [hunk 2]
```diff
@@ -33,16 +33,15 @@ SensorReader::SensorReader(int32_t type) {
 	this->sensorManager = getSensorManager();
 	this->sensor = ASensorManager_getDefaultSensor(this->sensorManager, type);
 	ALOGI("Using sensor: %s", ASensor_getName(this->sensor));
-	this->minDelay = ASensor_getMinDelay(this->sensor);
+	int minDelayMicroseconds = ASensor_getMinDelay(this->sensor);
```
Comment (sebastianbaginski) [inline vidhance/sensors/SensorReader.cpp]: `const int` :dancers:

**Comment by sebastianbaginski** [comment]
Looks ~~const~~ good.

File: vidhance/sensors/SensorReader.cpp [hunk 3]
```diff
@@ -82,7 +82,7 @@ std::vector<ASensorEvent> SensorReader::getEvents(vidhance_date_time_t timestamp
 	}
 	return result;
 }
-vidhance_time_span_t SensorReader::getSamplePeriod() {
+const vidhance_time_span_t SensorReader::getSamplePeriod() {
```
Comment (sebastianbaginski) [inline vidhance/sensors/SensorReader.cpp]: This method is still not const, here the modifier applies to return type, not the method type. What I meant was `vidhance_time_span_t SensorReader::getSamplePeriod() const {` ;)

Comment (emilwestergren) [inline vidhance/sensors/SensorReader.cpp]: Oh now I get it.

**Pull request merged**